content/millennium/r4/medications/medication_request.md

```diff
@@ -0,0 +1,144 @@
+    * [Validity Period](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.validityPeriod ){:target="_blank"}
+    * [Number of Refills](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.numberOfRepeatsAllowed){:target="_blank"}
+    * [Dispense Quantity](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.quantity){:target="_blank"}
+    * [Expected Days Supply per Dispense](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.expectedSupplyDuration){:target="_blank"}
+    * [Intended Dispenser](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.performer){:target="_blank"}
+* [Substitution Allowed Boolean](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.substitution.allowed_x_){:target="_blank"}
+* [Prior Prescription](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.priorPrescription){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:medication_request, :r4) %>
+
+### Contained Medication Bindings
+
+<%= terminology_table(:medication_request_contained_medication, :r4) %>
+
+## Search
+
+Search for MedicationRequests that meet supplied query parameters:
+
+    GET /MedicationRequest?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name      | Required?          | Type          | Description
+-----------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
+ `patient`                | This, or `_id`     | [`reference`] | The specific patient to return MedicationRequests for. Example: `12345`
+ `status`                 | N                  | [`token`]     | The status of the medication, may be a list separated by commas. Example: `active,completed`
+ `-timing-boundsPeriod`   | N                  | [`token`]     | The date-time which should fall within the dosageInstruction.timing.repeat.boundsPeriod the medication should be given to the patient. Must be prefixed by ‘ge’. Example: `ge2014-05-19T20:54:02.000Z`
+ `_lastUpdated`           | N                  | [`date`]      | An explicit or implied date-time range within which the most recent clinically relevant update was made to the medication. Must be prefixed by ‘ge’ or ‘le’. Example: `ge2014-05-19T20:54:02.000Z`
+ `_count`                 | N                  | [`number`]    | The maximum number of results to include in a page. Example: `50`
+
+ * The `_lastUpdated` and `-timing-boundsPeriod` parameter may be provided:
```

Comment:
Need to clarify on this.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-91cabf44fbcf0c5aaa861d340e57d246R84